projects/objectnav_baselines/experiments/objectnav_mixin_ddppo.py

```diff
@@ -34,13 +125,17 @@ def training_pipeline(self, **kwargs):
             update_repeats=update_repeats,
             max_grad_norm=max_grad_norm,
             num_steps=num_steps,
-            named_losses={"ppo_loss": PPO(**PPOConfig)},
+            named_losses={key: val[0] for key, val in named_losses.items()},
             gamma=gamma,
             use_gae=use_gae,
             gae_lambda=gae_lambda,
-            advance_scene_rollout_period=self.ADVANCE_SCENE_ROLLOUT_PERIOD,
+            advance_scene_rollout_period=cls.ADVANCE_SCENE_ROLLOUT_PERIOD,
```

Comment:
I'd keep this as `self` rather than `cls`. `self` will default to using `cls` if there is no instance-level definition of the this value but the reverse is not true if someone decides to define `ADVANCE_SCENE_ROLLOUT_PERIOD` at the instance level.